Docs/Configuration.tex

```diff
@@ -1882,11 +1882,10 @@ \subsection{Properties}\label{kernelprops}
 
   \emph{Note 2}: To track the dependency order, inspect the \texttt{OSBundleLibraries}
   key in the \texttt{Info.plist} file of the kext being added. Any kext included
-  in the \texttt{OSBundleLibraries} key is a dependency that must precede
-  the kext being added.
+  under the key is a dependency that must appear before the kext being added.
 
-  \emph{Note 3}: Kexts may have inner kexts (\texttt{Plugins}) included
-  in the bundle and each such plugin must be added separately.
+  \emph{Note 3}: Kexts may have inner kexts (\texttt{Plugins}) included in the bundle.
+  Each such plugin must be added separately and must appear after the host kext.
```

Comment:
This is not correct. Some kexts put plugins as their dependencies. Do not ask me why.

Reply:
Fixed